duqtools/cleanup.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import logging
+import shutil
+from typing import List
+
+import yaml
+from pydantic import DirectoryPath
+
+from duqtools.ids import IDSOperation, ImasLocation
+
+from .basemodel import BaseModel
+from .config import cfg
+
+logger = logging.getLogger(__name__)
+
+
+class Run(BaseModel):
+    dirname: DirectoryPath
+    data: 'ImasLocation'
+    operations: 'List[IDSOperation]'
+
+
+class Runs(BaseModel):
+    __root__: List[Run] = []
+
+    def __iter__(self):
+        yield from self.__root__
```

Comment:
This is probably the cause of the circular imports?
I moved all the config to the config.py file for that reason, but, they could also be moved to a separate file in `config/cleanup.py` for example, this would definitely get rid of the circular imports (that only occur when config declarations are present in the files which contain functions that use the cfg object

Reply:
Yeah, that's exactly the cause. I tried a couple of things, but couldn't find a way that I was happy with.